submissions/kriLEBED/html_css_popup/index.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Ubuntu&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css" integrity="sha512-NhSC1YmyruXifcj/KFRWoC561YpHpc5Jtzgvbuzx5VozKpWvQ+4nXhPdFgmx8xqexRcpAglTj9sIBWINXa8x5w==" crossorigin="anonymous" />
+    <title>My popup</title>
+</head>
+<body>
+    <header>
+        <div class="wrapper">
+            <nav>
+                <ul class="nav_menu">
+                    <li class="menu_items" tabindex="1"><a href="#" class="menu-link">Mail</a></li>
```

Comment:
You don't need a `tabindex` attribute here, because link(`a`) by default can receive focus, no need to change this behavior in this case.)